### PR TITLE
feat(npm-global): test global install

### LINKED ISSUE #1587: Add tests for global prisma installation
Tests if `npm install -g prisma` succeeds and tests `prisma --version`

### PR DESCRIPTION
**Pull request by millsp**
Fixes https://github.com/prisma/e2e-tests/issues/1587

### CODE REVIEW

**Comment by Jolg42** [comment]
Is this ready for review (I see it's a draft)

**Comment by millsp** [comment]
@Jolg42 I'm unsure what we said with @timsuchanek, I think I should test the `client` version since the `cli` version is always pulled from its `package.json` - the test won't fail. Should I install `@prisma/client` globally too, and test its version?

**Comment by Jolg42** [comment]
I don't think there is a use-case for testing a global install of `@prisma/client`.

Testing a global of the CLI only, is what I was thinking about.

**Comment by millsp** [comment]
Ok then you shall review :+1:

**Comment by timsuchanek** [comment]
:+1: then you can remove the "draft" mode. The only thing I would like to check, is if `prisma --version` doesn't error.

File: packagers/npm-global/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# npm package manager
+
+Tests `npm`.
```
Comment (janpio) [inline packagers/npm-global/README.md:3]: Missing update on test README

Comment (Jolg42) [reply packagers/npm-global/README.md:3]: Changed it to `Tests global install of `prisma` package with `npm`.`

**Pull request merged**